godotopenxrmeta/src/main/cpp/export/meta_export_plugin.h

```diff
@@ -52,6 +52,10 @@ static const int HAND_TRACKING_REQUIRED_VALUE = 2;
 static const int HAND_TRACKING_FREQUENCY_LOW_VALUE = 0;
 static const int HAND_TRACKING_FREQUENCY_HIGH_VALUE = 1;
 
+static const int BOUNDARY_ENABLED_VALUE = 0;
+static const int BOUNDARY_DISABLED_VALUE = 1;
+static const int BOUNDARY_CONTEXTUAL_VALUE = 2;
```

Comment:
Do these constants end up being used? I don't see them referenced in the code.

Reply:
removed in latest update, leftover from when I split the export logic from my other pull request that added boundary-less mode and other features